app/forms/support/accredited_provider_form.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module Support
+  class AccreditedProviderForm < Form
```

Comment:
I removed this in an earlier PR as I feel we should just be using the same form for support/publish. It's essentially the same behaviour? Couldn't we use the existing one https://github.com/DFE-Digital/publish-teacher-training/blob/main/app/forms/accredited_provider_form.rb

Reply:
Done